emulators/melonds/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           qt5 1.0
+
+github.setup        Arisotura melonDS 0.9.2
+revision            0
+
+name                melonds
+platforms           darwin
+license             GPL-3
+maintainers         {gmail.com:audvare @Tatsh} openmaintainer
+categories          emulators games
+description         Nintendo DS emulator.
+long_description    {*}${description}
+
+checksums           rmd160  87d6585506c8abdae6bf7417708e1b30578b1927 \
+                    sha256  d20ce23725e63be5c60d42c87ed504015aa57ddf57c5161302c98e1837e9acae \
+                    size    1331032
+
+depends_lib         port:gettext \
+                    port:glib2 \
+                    port:libarchive \
+                    port:libepoxy \
+                    port:libiconv \
+                    port:libsdl2 \
+                    port:libslirp
+
+qt5.min_version     5.12
+
+destroot {
+    copy ${workpath}/build/melonDS.app ${destroot}${applications_dir}
+}
+
+if {${os.major} < 18} {
+    post-install {
+        ui_warn "${name} @${version} is only supported on macOS 10.14 or later and may not work properly on your OS version."
```

Comment:
Are you saying it doesn’t build at all, or builds but the results might be unstable, or ??

Reply:
If it does build, all bets are off because upstream only wants 10.14 or later. That's why I put this in `post-install`.

Reply:
well ok, lets se what the reasons are in the buildbots then.